x/ibc/light-clients/07-tendermint/types/upgrade.go

```diff
@@ -40,6 +42,13 @@ func (cs ClientState) VerifyUpgradeAndUpdateState(
 			cs.GetLatestHeight().GetVersionNumber(), upgradeHeight.GetVersionNumber())
 	}
 
+	// UpgradeHeight must be greater than or equal to current client state height
+	if cs.GetLatestHeight().GetVersionHeight() > upgradeHeight.GetVersionHeight() {
```

Comment:
I wasn't sure if they could be equal but it sounds logical to me that the latest client state must be less than or equal to the upgrade height?